src/amo/reducers/collections.js

```diff
@@ -1308,6 +1322,20 @@ const reducer = (
       return unloadUserCollections({ state, username });
     }
 
+    case REMOVE_ADDON_FROM_COLLECTION: {
+      return {
+        ...state,
+        hasAddonBeenRemoved: false,
+      };
+    }
+
+    case ADDON_REMOVED_FROM_COLLECTION: {
+      return {
+        ...state,
+        hasAddonBeenRemoved: true,
```

Comment:
I don't know if this matters but should this get reset back to false? I was looking at redux and saw after it was removed it just stayed in this true state? just trying to understand : )

Reply:
I'm not sure about this. I can see what you mean - it does seem like it should get reset at some point, but I'm not sure at what point that should be. Would it be after the notice fades out? Having the code as it is doesn't seem to cause any problems, so I think it's okay, but let's see if @kumar303 has any concerns about this as well.

Reply:
Actually, this crossed my mind when reviewing too but I decided that it's not necessary for the current behavior and will only add more complexity to the reducer and the tests. I don't think it's worth it.

Reply:
> I think it's a little awkward to track ephemeral data such as notifications in Redux.

Actually, duh, it isn't awkward at all. You just dispatch one action to show the notification and another action from `setTimeout` to hide it. 

I left a detailed review comment!